README.md

```diff
@@ -1,11 +1,121 @@
+-   Xray Server/DC integration with Jenkins: <https://confluence.xpand-it.com/display/public/XRAY/Integration+with+Jenkins>
+
+You can also check the source code of the plugin here: <https://github.com/jenkinsci/xray-connector-plugin>
+  
+
+### Features
+
+-   Integration with both Xray in Jira Cloud and Jira Server/Data Center
+-   Export Cucumber features files from Xray
+-   Import Cucumber feature files to Xray
+-   Import your test results in any format supported by Xray
+
+### Requirements
+
+-   Jenkins 2.60.3 or above
+-   At least one Jira instance (Server, Data Center and/or Cloud) with
+    the
+    [Xray](https://marketplace.atlassian.com/apps/1211769/xray-test-management-for-jira)
+    installed.
+
+### Configuration
+
+Xray for Jenkins is configured in the global settings configuration page:
+
+**Manage Jenkins** \> **Configure System \> Xray configuration**.
+<br/><br/>
+
+To add a new Jira instance connection, you need to specify some
+properties:
+
+1.  **Configuration alias:** the name you want to give to this
+    configuration
+2.  **Hosting:** Hosting (instance type) in this case Cloud/Server/Data
+    Center
+3.  **(Server/Data Center only) Server Address**: The address of the
+    Jira Server where Xray is running
+4.  Credentials:  
+    1.  Use the **Jenkins Credentials Plugin** to set the
+        username/password (if you are using a Server/Data Center
+        instance) or an API key/secret (if you are using an Cloud
+        instance)
+    2.  Make sure that the user you are using have the following
+        permissions in the projects where you want to import the results
+        and import/export feature files: **View, Edit, Create**
+
+Note: the Configuration ID is not editable. This value can be used in
+the pipelines scripts.
+<br/><br/>
+After setting the instances, you can test your connection to Jira, by
+using the and ***Test Connection*** button.
+When you are done configuring your Jira instances, don't forget to
+save the changes.
+![](https://user-images.githubusercontent.com/11166836/72537278-c1151400-3873-11ea-80df-f10b2d993305.png)
+
+### Job Tasks
+
+Currently, we support **Freestyle**, **Pipeline**, **Multi-configuration
+Projects** and **Maven projects**.
+
+This plugin provides 2 new Build tasks:
+
+The ***Xray: Cucumber Features Export Task*** and the ***Xray: Cucumber
+Features Import Task***
+
+Both tasks can be used while executing Cucumber tests: In a
+typical [Cucumber
+Workflow](http://confluence.xpand-addons.com/display/XRAY/Testing+with+Cucumber),
+after having created a Cucumber project and the Cucumber tests specified
+in Jira, you may want to have a project that **exports** the features
+from Jira, executes the automated tests on a CI environment and then
+**imports** back its results.
+
+For this scenario, the Jenkins project would be configured with a set of
+tasks responsible for:
+
```

Comment:
1.  Pulling the Cucumber project
2.  **Exporting Cucumber features from Jira to your Cucumber project**
3.  Executing the tests in the CI environment
4.  **Importing the execution results back to Jira**

Here you don't talk about the "Cucumber Features Import Task", but in the below screenshot you're showing it. Is the screenshot in the right place? 

Perhaps the screenshot is supposed to be under the phrase "The ***Xray: Cucumber Features Export Task*** and the ***Xray: Cucumber Features Import Task***"?